src/i18n/i18n.types.ts

```diff
@@ -0,0 +1,4 @@
+export type localizedLanguagesStrings = {
```

Comment:
Why is the type in camel case? Also, do we need the Suffix Strings here?

Reply:
renamed to LocalizedLanguages